### ISSUE TITLE
Adding more paradigms for verbs

### ISSUE THREAD

**Issue by drdhaval2785** [mbykov/morpheus.sa.js#7]
Right now the verb paradigms are very limited, mostly to laT and laN lakAras.
For 10th gaNa, there is none.
Please expand them.

**Comment by mbykov** [comment]
Of course, I'll do that. For verb paradigms please look also morpheus.sa.js/lib/gana.js file. Tenth paradigm I am trying to separate from 4 or 6 after getting result from DB (possible dicts) in /lib/couch.js file. Try 'make gana g=ten_' for tests
